src/components/Map/index.js

```diff
@@ -113,6 +115,7 @@ const MapView = ({coordinates = null}) => {
     if (!coordinates.filter(value => isFinite(value)).length === 2) return;
     map.setView(coordinates);
   }, [map, coordinates]);
+
```

Comment:
so the previous PR removed the blank line, and this one adds it back? :stuck_out_tongue: 

Please avoid these kind of cosmetic changes that only clutter the diff :wink: